content/applications/hr/time_off.rst

```diff
@@ -463,54 +517,117 @@ Request allocation
+  requested.
 - :guilabel:`Add a reason...`: enter a description for the allocation request. This should include
   any details that managers and approvers may need in order to approve the request.
 
-When the form is complete, click the :guilabel:`Save` button to save the information, and submit the
-request.
+If the request was created from the :guilabel:`dashboard`, click the :guilabel:`Save & Close` button
+to save the information and submit the request. If the form was completed from the :guilabel:`My
+Allocations` list view, the information is automatically saved as it is entered. However, the form
+can be saved manually at any time by clicking the :guilabel:`Save manually` option, represented by a
+:guilabel:`cloud with an upwards arrow inside` icon.
 
 .. image:: time_off/allocation-request.png
    :align: center
    :alt: An allocation request form filled out for an employee requesting an additional week of
          sick time.
 
-Approvals
-=========
+.. _time_off/approvals:
+
+Management
+==========
 
-Most requests for time off and allocations need to go through the approval process, prior to the
-time off being allocated, and then granted to an employee. Requests either need one or two
-approvals, depending on how the specific type of time off is configured.
+Time off and allocation requests typically undergo an approval process before being granted.
+Requests either need one or two approvals, if any, depending on how the specific type of time off is
+configured. These are all housed under the :guilabel:`Management` section of the *Time Off*
+application.
 
-Only users who can approve allocation and time off requests have the :guilabel:`Approvals` section
+Only users who can approve allocation and time off requests have the :guilabel:`Management` section
 visible in the *Time Off* application.
 
-Approve allocations
--------------------
+.. _time_off/manage-time-off:
+
+Manage time off
+---------------
+
+To view time off requests that need approval, navigate to :menuselection:`Time Off app -->
+Management --> Time Off`. The only time off requests visible on this list are for employees the user
+has either :guilabel:`Time Off Officer` or :guilabel:`Administrator` access rights for the *Time
+Off* application.
+
+The default filter in the :guilabel:`Time Off` list is :guilabel:`Waiting For Me`. This only
+presents time off requests that need to be approved for current employees on the user's team, with a
+status of either :guilabel:`To Approve` or :guilabel:`Second Approval`.
 
-To view allocations that need approval, navigate to :menuselection:`Time Off app --> Approvals -->
+The left side of the screen has various grouping options to narrow down the presented time off
+requests. Since only time off requests that need to be approved are shown, the only status options
+are :guilabel:`All`, :guilabel:`To Approve`, and :guilabel:`Second Approval`. The various
```

Comment:
This ONLY appears if there are items that have that particular status for the signed-in user. In the runbots, there are never any requests with a "to approve" status- so I had to make one in a DB and see that it appears. So that's why you won't see any in a runbot. But you can also reject a request, then set it to draft and force it to have a 'to approve' status- THEN you can see it! I will add a note that if there aren't requests for the specific status (or dept), it's not seen.